x/mm2/types/params.go

```diff
@@ -0,0 +1,59 @@
+package types
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+)
+
+const (
+	// DefaultVersion is the default value for the Version Param.
+	DefaultVersion = 0
```

Comment:
following up on the other point, we should just hard code this to be the tag when the mm is ready.

Reply:
presumably `v1.0.0`

Reply:
What do you mean by this?

Reply:
the release we cut with the mm is probably going to be v1.0.0